### DISCUSSION TITLE
Set picom to NOT use tranparency

### DISCUSSION THREAD

**Discussion by se7en-x230** [yshui/picom#1024] [Q&A]
I would want to setup picom with rounded corners and some of the window fading effects BUT NO transparency.
I fail to understand how to archive that.

OS: Arch
WM: DWM
Term: st

**Comment by absolutelynothelix** [comment]
picom itself doesn't add any kind of transparency (unless it's configured to do so with, for example, the `opacity-rule` option or the `picom-trans` command line utility), it just allows windows that request transparency to be transparent. you have to configure desired applications to not request transparency.

**Comment by lystymoi** [comment]
Check your ~/.config/picom.conf for keyword "opacity".

For example setting
 ```
popup_menu = { opacity = 1; }
dropdown_menu = { opacity = 1; }
```
or completely removing those lines disables opacity for certain windows.
Some distributions could have preconfigured opacity for other windows.
